stackstorm-compose/README.md

```diff
@@ -0,0 +1,305 @@
+# StackStorm in Docker Compose
+
+This docker-compose is provided as a way to allow someone to "get up and running" quickly with StackStorm using Docker. It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.
+
+## TL;DR
+
+```shell
+git clone git@github.com:stackstorm/st2-dockerfiles
+cd st2-docker/stackstorm-compose
+docker-compose up -d
+docker-compose run st2api st2ctl reload --register-all  # you should only have to do this the first time
+docker-compose run st2client bash  # this gives you access to the st2 command line
+```
+
+Open `http://localhost:8000` in your browser. StackStorm Username/Password by default is: st2admin/Ch@ngeMe.
+
+## Usage
+
+### Prerequisites
+
+- Docker Engine 18.09+
+- Docker Compose 1.12+
+
+### Compose Configuration
+
+The image version, exposed ports, and "packs.dev" directory is configurable with environment variables.
+
+- **ST2_VERSION** this is the tag at the end of the docker image (ie: stackstorm/st2api:v3.3dev)
+- **ST2_IMAGE_REPO** The image or path to the images. Default is "stackstorm/".  You may change this is using the Enterprise version or a private docker repository.
+- **ST2_EXPOSE_HTTP**  Port to expose st2web port 80 on.  Default is `8000`, and you amy want to do `0.0.0.0:8000` to expose on all interfaces.
```

Comment:
1) Small typo `amy` vs. `may`.
2) I think the statement here is a bit misleading. With the current configuration in docker-compose.yml the exposed port **is** already bound to 0.0.0.0. In my opinion we should either use a default like 127.0.0.1 or state in the README that st2web is be exposed on a public interface.

Reply:
Good catch, I meant to have docker-compose default to localhost.